### PR TITLE
Customize RUDA Investment

### PR DESCRIPTION
**Pull request by erickyvand**

### CODE REVIEW

**Comment by fridolinho** [comment]
1. let try this as a main color : #03b8aa
2. Change the title to their business name
3. Remove street
4. remove twitter, instagram, linkedin and youtube since they are not available

**Comment by erickyvand** [comment]
> 3\. Remove

@fridolinho which title are you mentioning? because on my side I have the title as their business name

**Comment by fridolinho** [comment]
> > 3. Remove
> 
> @fridolinho which title are you mentioning? because on my side I have the title as their business name

akadomorw

File: ruda/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+			href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css"
+			rel="stylesheet"
+			integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1"
+			crossorigin="anonymous"
+		/>
+		<link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
+		<script src="https://use.fontawesome.com/c28c4f7255.js"></script>
+		<link rel="stylesheet" href="style.css" />
+		<link rel="shortcut icon" href="./images/logo.jpeg" type="image/x-icon">
+		<title>AkadomoRW</title>
```
Comment (fridolinho) [inline ruda/index.html:16]: RUDA Investment

**Review by fridolinho — CHANGES_REQUESTED** [review]

File: ruda/components/footer.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div class="common-bg-color text-center p-4">
+	<span class="text-white">Kitech Copyright &copy; 2021</span>
```
Comment (fridolinho) [inline ruda/components/footer.html:2]: RUDA Investment  &copy; 2021

**Review by fridolinho — APPROVED** [review]

**Pull request merged**